src/index.html

```diff
@@ -11,7 +11,9 @@
       href="https://fonts.gstatic.com"
       crossorigin="use-credentials"
     >
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap"
+      rel="stylesheet">
```

Comment:
```suggestion
    <link
      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap"
      rel="stylesheet"
    >
```
[code style](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md#:~:text=%5BCODE%20STYLE%5D%20%2D%20If%20the%20HTML%2Delement%20has%20long%20attribute%20values%20or%20number%20of%20attributes%20is%20more%20than%203%20%2D%20start%20each%20one%2C%20including%20the%20first%2C%20on%20the%20new%20line%20with%202%2Dspace%20indentation%20related%20to%20tag.%20Tag%E2%80%99s%20closing%20bracket%20should%20be%20on%20the%20same%20level%20as%20opening%20one.)